src/pages/docs/liveobjects/concepts/path-object.mdx

```diff
@@ -0,0 +1,506 @@
+```
+</Code>
+
+### at() method <a id="at"/>
+
+The `at(pathString)` method navigates using a dot-separated path string, which is useful for deeply nested paths:
+
+<Code>
+```javascript
+// Navigate using a path string
+const color = myObject.at('settings.theme.color');
+
+// Equivalent to chained get() calls
+const color = myObject.get('settings').get('theme').get('color');
+```
+</Code>
+
+<Aside data-type='note'>
+When using `at()` with a path string, dots (`.`) are treated as path separators. To include a literal dot in a key name, escape it with a backslash (`\.`). Malformed path strings (such as unescaped special characters) may result in unexpected path resolution.
+</Aside>
+
+### path() method <a id="path"/>
+
+The `path()` method returns the string representation of the current path:
+
+<Code>
+```javascript
+const settings = myObject.get('settings');
+console.log(settings.path()); // "settings"
+
+// Paths with dots in keys are escaped
+const value = myObject.get('config').get('api.endpoint'); // Key contains a dot
+console.log(value.path()); // "config.api\.endpoint"
+```
+</Code>
+
+The path string format is compatible with the REST API path format.
+
+## Access values <a id="access"/>
+
+PathObject provides several methods to access the underlying data.
+
+### value() method <a id="value"/>
+
+For primitive values and counters, use the `value()` method to get the current value:
+
+<Code>
+```javascript
+const visits = myObject.get('visits');
+console.log('Visits:', visits.value()); // e.g., 42
+
+const username = myObject.get('username');
+console.log('Username:', username.value()); // e.g., "alice"
+```
+</Code>
+
+If the path doesn't resolve to an entry, `value()` returns `undefined`.
+
+### instance() method <a id="instance"/>
+
+To get the specific [object instance](/docs/liveobjects/concepts/instance) at a path, use the `instance()` method:
+
+<Code>
+```javascript
+const counterInstance = myObject.get('visits').instance();
+
+if (counterInstance) {
+  // Work with the specific counter instance
+  console.log('Counter ID:', counterInstance.id());
+  await counterInstance.increment(1);
+}
+```
+</Code>
+
+The `instance()` method returns `undefined` if no object exists at that path. See the [Instance documentation](/docs/liveobjects/concepts/instance) for more details on working with object instances.
+
+### compact() method <a id="compact"/>
+
+The `compact()` method returns a JavaScript object representation of the data at a path:
+
+<Code>
+```javascript
+const settings = myObject.get('settings');
+const json = settings.compact();
+console.log(json);
+// {
+//   theme: { color: 'dark', fontSize: 14 },
+//   notifications: true
+// }
+
+const counter = myObject.get('visits');
+console.log(counter.compact()); // Just the number, e.g., 42
+```
+</Code>
+
+#### Binary values <a id="compact-binary"/>
+
+Binary values (such as `ArrayBuffer`) are returned as base64-encoded strings:
```

Comment:
I think we need to implement what we discussed the other day - i.e. the compact method should either return a JSON serializable value which is identical to that returned by REST, or it should return a JS object with cyclic references and ArrayBuffer binary values.

Let me know if we need to sync on this. Changing the behaviour would be a breaking change post release, so I would suggest we tag the method experimental in the release (and label it as such here) before we do so.